Prepare Getting Started with Popup Guide (#2137)
DevExpress/devextreme-documentation: concepts/05 UI Components/Popup/00 Getting Started with Popup/07 Add Scrolling.md

Users can scroll popup's content. To enable this functionality, wrap the content into the [ScrollView](/Documentation/Guide/UI_Components/ScrollView/Overview/) component and set its **height** and **width** to be 100% of the popup content area:

---
##### jQuery

    <!-- tab: index.js -->
    $(function () {
        $("#popup").dxPopup({
            contentTemplate: () => {
                const content = $("<div />");
                content.dxScrollView({
                    height: "100%",
                    width: "100%"
                });
                return content;
            },
            // ...
        });
    });
 
##### Angular

    <!-- tab: app.component.html -->

>     <dx-popup...>

Reviewer: ```suggestion
    <dx-popup ... >
```

        <div *dxTemplate="let data of 'content'">
            <dx-scroll-view 
                width="100%"
                height="100%">

>                 <!--...-->

Reviewer: ```suggestion
                <!-- ... -->
```

            </dx-scroll-view>
        </div>
    </dx-popup>
    <!-- ... -->

>     <!-- tab: app.component.ts -->
>     import { Component } from '@angular/core';
> 
>     @Component({
>         selector: 'app-root',
>         templateUrl: './app.component.html',
>         styleUrls: ['./app.component.css']
>     })
> 
> 

Reviewer: ```suggestion
```

##### Vue

    <!-- tab: App.vue -->
    <template>
        <div id="app-container">

>             <DxPopup...>

Reviewer: ```suggestion
            <DxPopup ... >
```

                <template #content>
                <DxScrollView
                    height="100%"
                    width="100%">
                    <!--...-->
                </DxScrollView>

>                 </template>            

Reviewer: ```suggestion
                <template #content>
                    <DxScrollView
                        height="100%"
                        width="100%">
                        <!-- ... -->
                    </DxScrollView>
                </template>            
```

            </DxPopup>
            <!-- ... -->
        </div>
    </template>

    <script>
    // ...
    import { DxScrollView } from "devextreme-vue/scroll-view";

    export default {
        // ...
        components: {
            // ...
            DxScrollView
        }
    }
    </script>

##### React

    <!-- tab: App.js -->
    // ...
    import ScrollView from 'devextreme-react/scroll-view';

>         const renderContent = () => {

Reviewer: This function has 4 extra spaces from the left.

            return (

>                 <>

Reviewer: ```suggestion
                <React.Fragment>
```

                    <ScrollView height="100%" width="100%">

>                         // ...

Reviewer: ```suggestion
                        {/* ... */}
```

                    </ScrollView>

>                 </>

Reviewer: ```suggestion
                </React.Fragment>
```

            )
        };

    const App = () => {
        // ...
        return (
            <div className="App">
                <Popup

>                     width={500}
>                     height={500}/>

Reviewer: ```suggestion
                    contentRender={renderContent}
                />
```

> Users can scroll popup's content. To enable this functionality, configure the [scrollView](/Documentation/Guide/UI_Components/ScrollView/Overview/) and set its **height** and **width**. 

Reviewer: ```suggestion
Users can scroll popup's content. To enable this functionality, wrap the content into the [ScrollView](/Documentation/Guide/UI_Components/ScrollView/Overview/) component and set its **height** and **width** to be 100% of the popup content area:
```

---
##### jQuery

    <!-- tab: index.js -->
    $(function () {
        $("#popup").dxPopup({
            contentTemplate: () => {

>                 const content = $("<div />");

Reviewer: You need to show full content template to illustrate this case properly.

                content.dxScrollView({
                    height: "100%",
                    width: "100%"
                });
                return content;
            },

>         // ...

Reviewer: ```suggestion
            // ...
```

        });
        // ...
    });

>     // ...

Reviewer: ```suggestion
```

> Users can scroll popup's content. To enable this functionality, wrap the content into the [ScrollView](/Documentation/Guide/UI_Components/ScrollView/Overview/) component and set its **height** and **width** to be 100% of the popup content area:

Reviewer 2: ```suggestion
Users can scroll the Popup's content. To enable this functionality, wrap the content into the [ScrollView](/Documentation/Guide/UI_Components/ScrollView/Overview/) component and set its **height** and **width** to 100% of the Popup content area:
```
